elasticgraph-warehouse/sig/elastic_graph/warehouse/schema_definition/schema_artifact_manager_extension.rbs

```diff
@@ -0,0 +1,11 @@
+module ElasticGraph
+  module Warehouse
+    module SchemaDefinition
+      module SchemaArtifactManagerExtension : ::ElasticGraph::SchemaDefinition::SchemaArtifactManager
+        private
+
+        def artifacts_from_schema_def: () -> ::Array[::ElasticGraph::SchemaDefinition::SchemaArtifact[untyped]]
```

Comment:
```suggestion
```

No need for these as you aren't impacting any method signatures in `SchemaArtifactManagerExtension`--you're simply overriding `artifacts_from_schema_def` which already exists on the supertype's RBS definition.